src/containers/item-details/index.tsx

```diff
@@ -31,6 +56,7 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
             className="col-lg-7 col-md-12 col-sm-12"
             style={{ height: '100vh', position: 'sticky' }}
           >
+            <a href={itemPinataUrl}>click image</a>
```

Comment:
Don't forget to comment this. This is just for testing if the NFT Image URL is working fine (**itemPinataUrl** state)